kubernetes/README.md

```diff
@@ -0,0 +1,19 @@
+# Cronjob for snapshoting a vault running on Kubernetes
+
+This assumes kubernetes authentication backend is configured on vault.
+
+The container image being used in this cronjob, is authenticating to Vault with the kubernetes authentication backend, with its serviceaccount JWT.
+
+The role and policy being used must be created before hand and must be used by the cronjob.
```

Comment:
You're talking about a Vault role and policy here, not a Kubernetes role right? So there no `role.yaml` missing in the PR.

Reply:
Yes, it is about Vault role & policy.